apps/sms/locales/sms.en-US.properties

```diff
@@ -295,9 +295,9 @@ send-button-sim-indication.ariaLabel = using SIM {{n}}
 # Labels for screen reader
 attach-button.ariaLabel = Add attachment
 contact-photo.ariaLabel = Contact photo
-unread-thread.ariaLabel = Unread
-has-draft.ariaLabel = Has draft
-is-draft.ariaLabel = Draft
+unread-thread = Unread
+has-draft = Has draft
+is-draft = Draft
```

Comment:
I'd suggest to keep "ariaLabel" in the name; did you check if `formatValue("unread-thread.ariaLabel")` was working ? If not maybe we can ask this to our l20n colleagues?